apps/etterlatte-saksbehandling-ui/client/src/components/behandling/BehandlingRoutes.tsx

```diff
@@ -192,7 +200,7 @@ export function soeknadRoutes(behandling: IBehandlingReducer, lagVarselbrev: boo
 
   const boddEllerArbeidetUtlandet = behandling.boddEllerArbeidetUtlandet?.boddEllerArbeidetUtlandet ?? false
 
-  if (avslag && boddEllerArbeidetUtlandet) {
```

Comment:
Jeg ville heller kanskje endret den til 
`if(avslag && boddEllerArbeidetUtlandet && vilkårsvurdering.yrkesskade)` ref https://jira.adeo.no/browse/EY-3503

Reply:
evt en kombi for det jeg sa løser jo ikke problemet egentlig men burde kanskje komme i tillegg